packages/overlay/src/index.tsx

```diff
@@ -31,13 +30,6 @@ export async function onSevereEvent(cb: (count: number) => void) {
   on("severeEventCount", e => cb((e as CustomEvent).detail?.count ?? 1));
 }
 
-/**
- * Sentry telemetry configuration for Spotlight.
- */
-export function sentry(options: { injectIntoSDK?: boolean } = {}) {
-  return initTelemetry(options);
-}
```

Comment:
I think this (an related stuff like `if (telemetry.setup)` below) should stay? Didn't see it in any other file as a newly added change.

Reply:
That's not my point. The initTelemetry here is the telemetry we send to Sentry using Sentry SDK. Sorry btw I posted this comment earlier but it was left in "pending" state as part of a general review

Reply:
I got confused sorry. `initSentry` was somewhere else so resolving this comment.